src/Utils.vala

```diff
@@ -61,4 +61,64 @@ public class Mail.Utils {
     public static string escape_html_tags (string input) {
         return input.replace ("<", "&lt;").replace (">", "&gt;");
     }
+
+    private static string accel_to_string (string accel) {
```

Comment:
@danrabbit maybe add a comment that this should be removed and replaced with the Granite util once it's released? It'd be nice to not lose track of it. :smile: